src/services/api.js

```diff
@@ -25,23 +24,11 @@ api.interceptors.response.use(
       window.location = '/'
     }
 
-    const originalRequest = error.config
-
-    if (error.response.status === 401 && originalRequest.url.startsWith(TOKEN_URL)) {
+    if (error.response.status === 401) {
       removeTokens()
       return Promise.reject(error)
     }
 
-    if (error.response.status === 401 && !originalRequest._retry) {
-      originalRequest._retry = true
-
-      const success = await refreshToken()
-      if (!success) {
-        removeTokens()
-        return Promise.reject(error)
-      }
-      return axios(originalRequest)
-    }
```

Comment:
maybe we should comment this instead and add a //TODO: put back oauth refresh token